pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/monitoring.test.yaml

```diff
@@ -10,17 +14,27 @@ tests:
     verifyMonitoringContainerPortExposed(container(.daemonsets.collector; "collector")) | assertThat(not)
     .networkpolicys["collector-monitoring"] | assertThat(. == null)
     .networkpolicys["admission-control-monitoring"] | assertThat(. == null)
+    verifyMonitoringExposed(.services["scanner-v4-indexer"]) | assertThat(not)
+    verifyMonitoringContainerPortExposed(container(.deployments["scanner-v4-indexer"]; "indexer")) | assertThat(not)
+    .networkpolicys["scanner-v4-indexer-monitoring"] | assertThat(. == null)
 
 - name: monitoring should be exposed when enabled
   set:
     exposeMonitoring: true
+    scannerV4:
```

Comment:
I think this is not correct. For helmtest you have two ways of specifying Helm input values.

The first is by using `values`:

```
values:
  foo:
    bar: 42
```

The other one is using `set`:

```
set:
  foo.bar: 42
```

Different styles, I don't think you can mix them (and expect them to mean the same thing).

Reply:
interesting that the test still passed, though 🤔

Reply:
oh these were the default values anyway